libs/ui-lib/lib/cim/components/InfraEnv/AddHostDropdown.tsx

```diff
@@ -24,10 +45,12 @@ const AddHostDropdown = ({
   onCreateBMH,
   docVersion,
   onCreateBmcByYaml,
-  isBMPlatform,
+  provisioningConfigResult,
 }: AddHostDropdownProps) => {
   const [addModalType, setAddModalType] = React.useState<ModalType>(undefined);
   const [isKebabOpen, setIsKebabOpen] = React.useState(false);
+  const [provisioningConfig, provisioningConfigLoaded, provisioningConfigError] =
+    provisioningConfigResult || [null, true, undefined];
```

Comment:
why is this needed ? the type of `provisioningConfigResult` does not allow for undefined/null